src/main/webapp/app/tasks/status-form/status-form.component.ts

```diff
@@ -0,0 +1,89 @@
+import { CdkDrag, CdkDragDrop, CdkDropList, moveItemInArray } from '@angular/cdk/drag-drop';
+import { ChangeDetectionStrategy, Component, Inject, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { v4 as uuid } from 'uuid';
+
+import { TaskService } from '../task.service';
+import { Task, TaskList } from '../task';
+import { DataHelper } from 'app/shared/helpers';
+import { Status } from '../enums';
+
+@Component({
+  selector: 'tiny-status-form',
+  templateUrl: './status-form.component.html',
+  styleUrls: ['./status-form.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StatusFormComponent implements OnInit {
+
+  tasks$: Observable<TaskList[] | Task[]>;
```

Comment:
this should be mapped to a string array of status names in the component to prevent business logic in the template